### PR TITLE
Cleanup & Reorg startup scripts

### PR DESCRIPTION
**Pull request by ryanmerolle**
Related Issue: #669

## New Behavior

Reorder startup scrips & delete duplicate scripts

## Contrast to Current Behavior

Duplicate scripts run

## Discussion: Benefits and Drawbacks

remove duplicated scripts, and enforce script order

## Changes to the Wiki

N/A

## Proposed Release Note Entry

Startup Scripts reorganization & cleanup

## Double Check

* [x] I have read the comments and followed the PR template.
* [x] I have explained my PR according to the information in the comments.
* [x] My PR targets the `develop` branch.

### CODE REVIEW

**Comment by ryanmerolle** [comment]
mypy linting does not like the below:
```python
53:            else:
54:                query = {field: params.pop(assoc)}  # 290_ip_addresses.py:54: error: Dict entry 0 has incompatible type "Optional[str]": "Any"; expected "str": "Any"
55:                params[assoc] = model.objects.get(**query)  # 290_ip_addresses.py:56: error: Item "None" of "Optional[Any]" has no attribute "objects"
```

It seems I am triggering the linting because I moved the file.  I likely need some assistance with this failure.

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .initializers
 docker-compose.override.yml
 *.pem
+.DS_Store
```
Comment (cimnine) [inline .gitignore]: Please add this to your [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

**Review by cimnine — CHANGES_REQUESTED** [review]

**Comment by cimnine** [comment]
> It seems I am triggering the linting because I moved the file. I likely need some assistance with this failure.

I currently don't have capacity to look into it. But as it's not your fault, don't worry about it. We will be able to merge even with this inconvenience.

What troubles me more is that the test does not run through.
Did it work when you tried it locally? From memory, I believe you can invoke it like this:

```bash
# Build latest
./build.sh master

# Test latest
./test.sh latest
```

Tipp: I often commented out the section in the `test.sh` where it runs NetBox's own unit tests when I developed on the initializers. Otherwise the test goes on forever.

File: .gitignore (hunk 1 above)
Comment (ryanmerolle) [reply .gitignore]: Apologies.  Fair enough

**Comment by ryanmerolle** [comment]
I appreciate the hand holding on this @cimnine

**Review by cimnine — APPROVED** [review]

**Review by tobiasge — APPROVED** [review]

**Pull request merged**